tests/traits/test_with_duration.py

```diff
@@ -29,9 +30,9 @@ def assert_duration():
     duration += (24 * 3600 * 30)
     assert duration28 <= wait.get_duration() <= duration31
     wait.years(1)
-    duration_31_full_year = duration31 + (24 * 3600 * 365)
-    duration_28_leap_year = duration28 - (24 * 3600)
-    assert duration_28_leap_year <= wait.get_duration() <= duration_31_full_year
+    duration_31_leap_year = duration31 + (24 * 3600 * 366)
```

Comment:
The logic was screwed: a leap year is actually a year with an extra day (365 -> 366), not the other way around.